gpt4all-backend/llmodel.cpp

```diff
@@ -104,7 +104,7 @@ const std::vector<LLModel::Implementation> &LLModel::Implementation::implementat
 
                     // Add to list if model implementation
                     try {
-                        Dlhandle dl(p.string());
+                        Dlhandle dl(std::filesystem::absolute(p).string());
```

Comment:
you can undo the changes here

Reply:
Done. Still works with the undo, so you might have been right.